src/containers/createEvent/index.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react';
+import { Helmet } from 'react-helmet';
+import { connect, useDispatch } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import { Routes } from '../../App';
+import { UserAuthenticationReducerState } from '../../auth/ducks/types';
+import { ContentContainer } from '../../components';
+import { C4CState } from '../../store';
+import {
+  AsyncRequest,
+  AsyncRequestCompleted,
+  AsyncRequestFailed,
+  asyncRequestIsComplete,
+  AsyncRequestLoading,
+  AsyncRequestNotStarted,
+} from '../../utils/asyncRequest';
+import EventsForm from '../../components/EventsForm';
+import FormInitialText from '../../components/FormInitialText';
+import { Typography } from 'antd';
+import { createEvent } from './ducks/thunks';
+import { EventsFormData } from '../../components/EventsForm';
+import { encodeProfileFieldFile } from '../../utils/fileEncoding';
+
+interface CreateEventProps {
+  readonly tokens: UserAuthenticationReducerState['tokens'];
+}
+
+const { Title } = Typography;
+
+const CreateEventContainer: React.FC<CreateEventProps> = ({ tokens }) => {
+  const dispatch = useDispatch();
+  const history = useHistory();
+  const [createEventRequest, setCreateEventRequest] = useState<
+    AsyncRequest<void, any>
+  >(AsyncRequestNotStarted());
+
+  if (asyncRequestIsComplete(createEventRequest)) {
+    history.push(Routes.UPCOMING_EVENTS);
```

Comment:
❓ Do you think it might be better to route directly to the single event page instead?

Reply:
Yeah makes more sense to direct the user to the page of the event they just created.